### PR TITLE
Fix/unlocking staking rewards

### PR DESCRIPTION
**Pull request by Divya-Solulab**
Implements logic to unlock staking rewards based on meeting a daily minimum transaction threshold on the Optimism network.

### CODE REVIEW

**Review by 0xArdi — APPROVED** [review]

File: packages/valory/contracts/staking_activity_checker/contract.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+fingerprint_ignore_patterns: []
+contracts: []
+class_name: StakingActivityCheckerContract
+contract_interface_paths:
+  ethereum: build/StakingActivityChecker.json
+dependencies:
+  open-aea-ledger-ethereum:
+    version: ==1.55.0
+  open-aea-test-autonomy:
+    version: ==0.15.2
```
Comment (Adamantios) [inline packages/valory/contracts/staking_activity_checker/contract.yaml:21]: I would suggest to use the same version.

File: packages/valory/contracts/staking_token/contract.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+fingerprint_ignore_patterns: []
+contracts: []
+class_name: StakingTokenContract
+contract_interface_paths:
+  ethereum: build/StakingToken.json
+dependencies:
+  open-aea-ledger-ethereum:
+    version: ==1.55.0
+  open-aea-test-autonomy:
+    version: ==0.15.2
```
Comment (Adamantios) [inline packages/valory/contracts/staking_token/contract.yaml:21]: Same here.

File: packages/valory/skills/liquidity_trader_abci/models.py [hunk 3]
```diff
@@ -107,4 +117,29 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.chain_to_chain_id_mapping = json.loads(
             self._ensure("chain_to_chain_id_mapping", kwargs, str)
         )
-        super().__init__(*args, **kwargs)
```
Comment (0xArdi) [inline packages/valory/skills/liquidity_trader_abci/models.py:110]: here

Comment (Adamantios) [reply packages/valory/skills/liquidity_trader_abci/models.py:110]: Well spotted!

Comment (Divya-Solulab) [reply packages/valory/skills/liquidity_trader_abci/models.py:110]: Thanks for catching that! I've corrected the mistake @0xArdi

**Review by Adamantios — APPROVED** [review]

**Pull request merged**